src/Npgsql.OpenTelemetry/README.md

```diff
@@ -20,3 +20,42 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
 Once this is done, you should start seeing Npgsql trace data appearing in your application's console. At this point, you can look into exporting your trace data to a more useful destination: systems such as [Zipkin](https://zipkin.io/) or [Jaeger](https://www.jaegertracing.io/) can efficiently collect and store your data, and provide user interfaces for querying and exploring it.
 
 For more information, [visit the diagnostics documentation page](https://www.npgsql.org/doc/diagnostics.html).
+
+### Enrich
+
+This option allows one to enrich the activity with additional information from the `NpgsqlCommand`, or on any exception.
+The `Enrich` action is called only when `activity.IsAllDataRequested` is `true`.
+It contains the activity itself (which can be enriched), the name of the event, and either the `NpgsqlCommand` or an exception, depending on the event name:
+
+For event name "OnStartActivity", the actual object will be `NpgsqlCommand`.
+
+For event name "OnStopActivity", the actual object will be `NpgsqlCommand`.
+
+For event name "OnException", the actual object will be `Exception`.
```

Comment:
I know that you're following the HTTP implementation, but it seems problematic to not have the NpgsqlCommand when enriching OnException; the usefulness of just having an Exception seems limited. Any reason not to pass a tuple of both the command and the exception?

Reply:
It certainly could be a tuple, which I think would be nice enough to work with, e.g.:
```
switch (eventName, rawObject)
{
    case ("OnException", (NpgsqlCommand command, Exception exception)):
        activity.SetTag("stackTrace", exception.StackTrace);
        break;
}
```
I wonder though, if we are always going to be passing the command if we should just be explicit about the type in the action signature, and just add an optional exception parameter, e.g.:
```Action<Activity, string, NpgsqlCommand, Exception?>```

I'm not sure which I prefer - wdyt?

Reply:
Definitely possible, but I'm not sure I like getting an always-null parameter for Start/Stop. Also, users would have to use the bang operator on Exception for the OnException case (it's guaranteed to be non-null), which isn't amazing.

I guess I'm leaning towards the tuple...